docs/websocket/feedback.md

```diff
@@ -63,15 +63,15 @@ the following json is sent as a progressive result:
 ```json
 {
   "sentence": 0,
-  "feedback": true
+  "errors": 0
 }
 ```
 
 Name        | Type     | Description
 ------------|----------|------------
 feedback_id | `string` | The unique id of the feedback this results belongs to.
```

Comment:
Somewhat related: feedback_id is not shown in the response json example above.

Reply:
+1

Reply:
Ah, good catch, will make a pr for that as well. :wink:

Reply:
https://github.com/itslanguage/itslanguage-docs/pull/78